packages/@okta/vuepress-theme-prose/global-components/Login.vue

```diff
@@ -52,5 +45,17 @@
 </template>
 
 <script>
-export default {};
+export default {
+  computed: {
+    uris() {
+      const { uris } = this.$site.themeConfig;
+
+      return {
+        email: "https://login.okta.com/",
+        github: `${uris.baseUri}/sso/idps/${uris.idps.github}`,
+        google: `${uris.baseUri}/sso/idps/${uris.idps.google}`,
+      };
```

Comment:
Can we make a helper that builds these? Seems like they're the same in both login and signup

Reply:
I had put it off since it's only used two places but I think it's worthwhile to avoid making a mistake changing one URI but not the other. I've added a commit to address this